src/test/java/pages/FileUploadPage.java

```diff
@@ -0,0 +1,59 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class FileUploadPage extends BasePage{
+    public final String FILE_UPLOAD_URL = "http://the-internet.herokuapp.com/upload";
+    public final String FILE_PATH = "C:\\Users\\dkwr2\\OneDrive\\Desktop\\test.txt";
+    public final By UPLOAD_PATH = By.id("file-upload");
+    public final By BUTTON_UPLOAD = By.id("file-submit");
+    public final By PATH_UPLOAD_DONE = By.id("uploaded-files");
+    public static final String ERROR = "error";
+    public final String EXP_FILE_NAME = "test.txt";
+    public FileUploadPage(WebDriver driver) {
+        super(driver);
+    }
+    /*
+    Open website http://the-internet.herokuapp.com/upload
+     */
+    @Override
+    public void openPage() {
+        driver.get(FILE_UPLOAD_URL);
+    }
+    /*
+    Enter path to file
+     */
+    public void uploadFilePath(){
```

Comment:
just uploadFile and pass file path as parameter in test

Reply:
done